docs/modules/ROOT/pages/tutorials/demo-app/gitea-forgejo.adoc

```diff
@@ -0,0 +1,98 @@
+= Application Demo: Gitea or Forgejo
+:app-link: https://www.gitea.io/[Gitea^] or https://forgejo.org/[Forgejo^]
+:app-logo: gitea.svg
+
+include::partial$app-demo-partials.adoc[tag=introduction]
+
+include::partial$app-demo-partials.adoc[tag=requirements]
+
+== About the Application
+
+> Gitea - Git with a cup of tea
+>
+> A painless self-hosted Git service.
+>
+> Gitea is a community managed lightweight code hosting solution written in Go. It's published under the MIT license.
+
+> Forgejo - Beyond coding. We forge.
+>
+> Forgejo is a self-hosted lightweight software forge.
+> Easy to install and low maintenance, it just does the job.
+> Brought to you by an inclusive community under the umbrella of Codeberg e. V., a democratic non-profit organization, Forgejo can be trusted to be exclusively Free Software. It's a "soft" fork of Gitea with a focus on scaling, federation and privacy.
+
+include::partial$app-demo-partials.adoc[tag=step1]
+
+== Step 2: Deploy the application
+
+To deploy the application we will use standard Kubernetes objects.
+Save the example YAML Kubernetes resources into a file and apply it with `oc apply -f <myfile.yaml>`.
+
+. First, we need a database:
++
+.Database ordering from the VSHN Application Catalog
+[source,yaml]
+----
+include::example$example-app-exo-mysql.yaml[]
+----
++
+This will create a MySQL DBaaS instance with default settings.
+See xref:appcat:ROOT:exoscale-dbaas/mysql/create.adoc[the AppCat docs] for more information.
+
+. Then we deploy the application:
++
+.Application deployment with all other needed resources
+[source,yaml]
+----
+include::example$example-app-gitea.yaml[]
+---
+include::example$example-app-service.yaml[]
+---
+include::example$example-app-pvc.yaml[]
+---
+include::example$example-app-route.yaml[]
+----
+
+. Now wait until your pod appears with the status "Running":
++
+[source,shell]
+--
+oc get pods --watch
+--
+
+. Last but not least retrieve the URL of your Gitea or Forgejo instance:
++
+[source,shell]
+--
+oc get route example-app -o jsonpath='{.spec.host}'
+--
+
+== Configure Gitea
```

Comment:
Isn't there a way to do this using the deployment (environment variables, secretes/config maps mounted as config file)?

Reply:
I tried that, and the Gitea image does not pay attention to the environment variables on first start. It shows a form to fill with various pieces of information, including the details of the default account, which must be created there (otherwise there's no "default" or "admin" account)

The official solution consists (apparently) in adding a ConfigMap with a full app.ini inside, and tbh I couldn't make it work. This solution is short, simple, not very complex, and gets the job done.

Reply:
I was expecting something like this.